frontend/issue.ts

```diff
@@ -216,6 +219,34 @@ export async function getIssueComments(
   });
 }
 
+export async function deleteIssueComments(
+  tx: WriteTransaction,
+  issueID: string
+): Promise<void> {
+  const entries = (await getIssueComments(tx, issueID)) || [];
+  await entries
```

Comment:
The return value of `map` is an array, so I don't think this `await` does anything. I think maybe you mean to put a `Promise.all()` around this like `await Promise.all(entries.map(...).map(...))`.

Reply:
yes, nice catch